### ISSUE TITLE
Feature Request - Add GUI entry for electricity cost

### ISSUE THREAD

**Issue by NoKwarl** [nemosminer/NemosMiner-v3.0-windows#399]
I noted the addition of electricity cost calculations in profit in the code updates, can this be added to the GUI? Can the profit calculation be broken down visually (for example - payout - (electricity + fee))? Can we also add a percentage allowance over software based electricity cost to account for power supply inefficiency/fans/other overhead?

**Comment by nemosminer** [comment]
Special Thanks to : hmts82

https://github.com/hmts82 

https://github.com/hmts82/NemosMiner-v2.5.2-windows/commit/a6c86691c17073f04e810fa922cc769727ed29b2

https://github.com/hmts82/NemosMiner-v2.5.2-windows/commit/b867328a0637d6fb50bb05b4edb27f30f8f63490

https://github.com/hmts82/NemosMiner-v2.5.2-windows/commit/cab5da8953be6cc4d3db81e6cce1175f4c534cbd

https://github.com/hmts82/NemosMiner-v2.5.2-windows/commit/a112fc800b0dcd9350b0c89cf91417d2fb0b88ce

**Comment by kingyem** [comment]
Just read about using nvidia-smi to display power usage of whole cards, can that be used for power consumption calculation instead of a constant value?

**Comment by dangi12012** [comment]
Yeah good idea. Benchmark must calculate avg hashrate + avg powerconsumption. The user has a field for kwh price and now the profit switching can be MUCH smarter.

**Comment by platinum4** [comment]
An example of code for this that works can be found here: https://github.com/UselessGuru/MultiPoolMiner/blob/master/Include.psm1

Unfortunately this fork is not well maintained and adding miners to it other than Ccminer based APIs doesn't work well; Bminer doesn't work on that fork so you have to use DSTM and the switching logic on it is a bit stupider.  Basically Lyra2z has a consumption factor of 0.60 if Keccak/Neoscrypt was a 1.0, x16r/x16s is 0.90 and Ethash is around that as well, there's really only those three tiers of consumption.  No real middle road but Lyra2z is not exactly half the usage so I overestimated at 0.60

**Comment by bhh7419** [comment]
I can provide a small sample if you wish

**Comment by bhh7419** [comment]
https://github.com/bhh7419/gpupowerman/tree/master

**Comment by platinum4** [comment]
You're way behind on this, MindMiner does it already

**Comment by nemosminer** [comment]
@UselessGuru Discord..

**Comment by nemosminer** [comment]
@UselessGuru https://discord.gg/yCqqtj

**Comment by nemosminer** [comment]
good things take time : )

**Comment by GuyTechie** [comment]
This would be awesome!

**Comment by Minerx117** [comment]
https://github.com/Minerx117/NemosMiner/issues/5
